### DISCUSSION TITLE
[BUG] Description - Menu bug with dropdown

### DISCUSSION THREAD

**Discussion by ant31zac** [bootscore/bootscore#481] [Q&A]
### Describe the bug

In menu configuration, i set this configuration:

- Prodotti
    - Distribuzione
         - Float
    - Trasformazione
         -  Laminati

But when i save menu and go on live site, the menu not showing the second dropdown

### Screenshots and Additional Info

![live_site](https://github.com/bootscore/bootscore/assets/102148811/0a71ea36-2471-4dc2-8017-c04d440aac11)
![menu_configuration](https://github.com/bootscore/bootscore/assets/102148811/eb053739-7f56-4228-adf6-b7ecb44ab1c4)

**Comment by crftwrk** [comment]
This won't work that way because Bootstrap does not support more than 2 level menus. You have to modify your `header.php` and the `nav-walker` to allow 3 or more levels.

Check https://github.com/orgs/bootscore/discussions/347 how to do this.

**Comment by ant31zac** [comment]
thanks so much for the fix

**Reply by crftwrk** [reply]
You‘re welcome. We leave this discussion open, because this is a often asked question.
